Database/Corrections/classicItemFixes.lua

```diff
@@ -344,6 +344,9 @@ function QuestieItemFixes:Load()
             [itemKeys.npcDrops] = {},
             [itemKeys.objectDrops] = {},
         },
+        [5233] = {
+            [itemKeys.npcDrops] = {1020,1021,1022,1023}, -- wotlkDB is wrong
```

Comment:
all of the fixes with this kind of comment are because wotlkDB is different from classicDB and wrong, whereas the classicDB is correct. I guess we should add them to wotlkfixes

Reply:
Yeah, I think so too. If you want to fix something exclusively for the wotlkDB than add the fixed to the wotlkQuestFixes.lua